src/router/index.ts

```diff
@@ -44,6 +44,15 @@ const routes: Array<RouteConfig> = [
     component: LrAlertDetails,
     props:true
   },
+  {
+    path: "/manage",
+    redirect: "s3Account"
```

Comment:
Why do you redirect this to S3 account page?

Reply:
As if now It is kept as default, after the authorization is implemented we would add logic to route on basis of type of user

Reply:
As discussed the code will go along with user admin page